sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Auth.java

```diff
@@ -147,6 +155,44 @@ public void onTokenRequiresRefresh(String homeAccountId) {
         }
     };
 
+    /**
+     * Init.
+     */
+    private Auth() {
+
+        /* Setup MSAL Logging. */
+        Logger.getInstance().setLogLevel(Logger.LogLevel.VERBOSE);
+        try {
+            Logger.getInstance().setExternalLogger(new ILoggerCallback() {
+
+                @Override
+                public void log(String tag, Logger.LogLevel logLevel, String message, boolean containsPII) {
+                    if (!containsPII) {
+                        String prefixedTag = LOG_TAG + TAG_DELIMITER + tag;
+                        switch (logLevel) {
```

Comment:
Code coverage (jacoco?) doesn't play well with switch statements 🤦‍♂. I'll use if/then/else.

Reply:
@clpolet That's known one - it tries to find test for generated hashes checks for strings